src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
It is a good practice to specify an access modifier for the class. In this case, 'public' would be appropriate for the ColorSupplier class.